src/reducer.js

```diff
@@ -565,6 +543,11 @@ export const onTokenFetchHistoryRequested = (state, action) => {
   };
 };
 
+export const onStartWalletNotStarted = (state) => ({
+  ...state,
+  walletStartState: WALLET_STATUS.NOT_STARTED,
+});
+
```

Comment:
Maybe we should change the `onStartWalletLock` method to also change state to `NOT_STARTED`, what do you think?

Reply:
I agree, refactored in 9fcb649c73fc0e5d62bfe787082f0dd8e98818cf